rfcs/0025-add-pytorch-tvm.md

```diff
@@ -0,0 +1,265 @@
+- Feature Name: PyTorchTVM
+- Start Date: 2021-08-24
+- RFC PR: [apache/tvm-rfcs#0025](https://github.com/apache/tvm-rfcs/pull/25)
+- GitHub Issue: TODO
+
+# Summary
+[summary]: #summary
+
+This RFC add a `PyTorchTVM` module to support: compile TorchScript to TVM and use accelerated module in PyTorch.
+
+To increase the TVM accessibility for PyTorch users, we propose `PyTorchTVM` module to support the following workflow:
+1. convert a torchscript module to tvm graph
```

Comment:
To clarify, is it an entire module (full graph) or a subgraph?

```suggestion
1. Convert a TorchScript module to TVM graph (Relay)
```

Reply:
A torchscript module contains other torchscript module. So users can choose a module at any level to do the conversion.

Reply:
what about

```suggestion
1. Convert a TorchScript module (or submodule) to TVM graph (Relay)
```